### PR TITLE
[auditcare][couch-to-sql]Remove code reading couch data

### PR DESCRIPTION
**Pull request by AmitPhulera**
## Product Description

https://dimagi-dev.atlassian.net/browse/SAAS-12241
## Technical Summary

The PR removes the code that splits queries between couch and SQL while reading Auditcare.
The check only existed in `export.py` only.

### Automated test coverage

There are already tests that verify that SQL is queried properly, the tests are modified/removed which queried couch.
### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by millerdev — COMMENTED** [review]
:+1: looks like there is more work to do here. Is that right?

**Comment by AmitPhulera** [comment]
@millerdev I think this should be it. I was waiting for the tests to pass. I did not find any other occurrence where we are querying Auditcare with the couch. 
I believe models and other code were removed before when we moved to SQL. Do you know any other place where we need to make the changes?

**Review by joxl — COMMENTED** [review]
This is fantastic.

File: corehq/apps/auditcare/utils/export.py [hunk 1]
```diff
@@ -179,52 +166,6 @@ def __iter__(self):
...
-
-def determine_sql_start_date(start_date):
-    fixed_start_date = get_fixed_start_date_for_sql()
-    if fixed_start_date and start_date < fixed_start_date:
-        return fixed_start_date
-    else:
-        return start_date
-
-
 def get_sql_start_date():
```
[the remark this answers is not in the document]
Comment (millerdev) [reply corehq/apps/auditcare/utils/export.py:169]: Maybe this should be moved over to corehq/apps/auditcare/utils/migration.py? It is no longer used in this module.

**Review by millerdev — APPROVED** [review]
I don't remember what made me think this was not done. I thought tests were failing when I last reviewed, but maybe not? In any case, this is great!

File: corehq/apps/auditcare/utils/export.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (AmitPhulera) [reply corehq/apps/auditcare/utils/export.py:169]: @joxl It is used by the migration code, so IMO `get_sql_start_date` would be much suited here.

I will move it to `migration.py`. Thanks for the suggestion @millerdev

**Review by millerdev — APPROVED** [review]

**Review by joxl — APPROVED** [review]

**Pull request merged**